modal/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
This file should be updated with all of the metrics that Modal plans to send into DAtadog

Reply:
Given that this is a log-only integration, for now at least, what should we add here?

Reply:
Ah okay,  in that case this should be left as-is